src/org/rascalmpl/util/maven/Artifact.java

```diff
@@ -151,6 +150,14 @@ private void calculateClassPath(Scope forScope, HashSet<WithoutVersion> alreadyI
         }
     }
 
+    private Artifact createSystemArtifact(Dependency d) {
+        var path = d.getSystemPath();
+        if (path == null) {
+            return Artifact.unresolved(d.getCoordinate(), messages.writer());
+        }
+        return new Artifact(d.getCoordinate(), null, Path.of(path), Collections.emptyList(), messages, null);
```

Comment:
I think we should still check if there is a pom, and make sure it doesn't have it's own depdnendcies?

Or what does maven say about system depdnendcies?

Reply:
Maven says:
"Dependencies with the scope system are not looked up in the Maven repository system. Instead the dependency element contains a systemPath pointing to a jar on the local file system"
So no transitive dependencies, just jars.